.ci/checker-framework.groovy

```diff
@@ -398,6 +396,21 @@ class CheckerFrameworkError implements Comparable<CheckerFrameworkError> {
     List<String> details
     String lineContent
     int lineNumber
+
+    /**
+     * Whether the error is unstable. Unstable errors in suppression list are not flagged as
```

Comment:
I do not think we need a list here, we can probably just eliminate for sentence and have next sentence lead into explanation in first list item.

Reply:
Removed the list.